fastly/sweeper_test.go

```diff
@@ -18,28 +21,30 @@ func TestMain(m *testing.M) {
 	resource.TestMain(m)
 }
 
-func sharedClientForRegion(region string) (*fastly.Client, error) {
+func sharedClientForRegion(region string) (*fastly.Client, diag.Diagnostics) {
 	if client, ok := sweeperClients[region]; ok {
 		return client, nil
 	}
 
-	if os.Getenv("FASTLY_API_KEY") == "" {
```

Comment:
Could you clarify the reason for removing this check?

Reply:
This check was removed because the same check is present in the fastly/config.go and this felt like the best place to focus.